doc/source/working_with_kiwi/xml_description.rst

```diff
@@ -307,6 +307,9 @@ build types and will be covered here:
   via the `-o` flag to :command:`mount` and are included in
   :file:`/etc/fstab`.
 
+- `fscreateoptions`: Specifies the filesystem options used to create the
```

Comment:
Please extend the docs about this feature a bit. If I didn't know what this was about, I wouldn't be able to figure out how to use it. 

I can write something for that if that helps.

Reply:
I would remark that the string provided here will just be part of the fs creation command call. This is not a comma separated value or anything like that. Probably adding here an straight forward example for ext4 as #1109 would make sense.

Reply:
fair enough, how about it now ?

Reply:
Very good, thank you!